services/device-registry-jdbc/src/test/java/org/eclipse/hono/deviceregistry/jdbc/impl/AbstractJdbcRegistryTest.java

```diff
@@ -153,6 +153,8 @@ void startDevices(final Vertx vertx) throws IOException, SQLException {
 
     private JdbcProperties resolveJdbcProperties() {
         final var jdbc = new JdbcProperties();
+        jdbc.setInitialPoolSize(0);
```

Comment:
Sorry for not noticing this earlier, but please update the copyright header of this file with the current year

Reply:
Done.